core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -34,11 +34,16 @@ import org.apache.kafka.common.protocol.{Errors, SecurityProtocol}
 import org.apache.kafka.common.security.JaasUtils
 import org.apache.kafka.common.serialization.StringDeserializer
 import org.apache.kafka.common.utils.Utils
-
 import scala.collection.JavaConverters._
 import scala.collection.{Set, mutable}
 
-object ConsumerGroupCommand {
+/**
+ *  To print debug message, add the following line to log4j.properties:
```

Comment:
Not sure we want this as the scaladoc. Besides, this seems like a general comment about a variety of tools and we probably don't want to document it separately for every tool. If we really need this, it seems like normal docs are a better place to add a generalized version of the note.